src/RolloutScorer/RolloutScorer/Commands.cs

```diff
@@ -69,7 +69,7 @@ public async Task<int> InvokeAsync(IEnumerable<string> arguments)
             _rolloutScorer.GithubConfig = StandardConfig.DefaultConfig.GithubConfig;
 
             // If they haven't told us to upload but they also haven't specified a repo & rollout start date, we need to throw
-            if (string.IsNullOrEmpty(_rolloutScorer.Repo) || (_rolloutScorer.RolloutStartDate == null))
+            if (string.IsNullOrEmpty(_rolloutScorer.Repo))
```

Comment:
Did you remove this on purpose?

Reply:
I did. RolloutStartDate is a DateTimeOffset, and, therefor, never null.